### ISSUE TITLE
milkweeds take 10 min to disappear making roads annoying to build

### ISSUE THREAD

**Issue by RainstormPw** [jasonrohrer/OneLife#986]
![image](https://github.com/jasonrohrer/OneLife/assets/150627362/0d3c323d-ea8d-4894-8f8e-a90a17e096ac)

**Comment by SoloAceMouse** [comment]
Yeah, it'd be nice if you could dig them up with a sharp stone or something to clear the debris like other wild plants.

**Comment by TwistedHBG** [comment]
They take 1 minute to decay unless you pick them when they're fruiting, in which case they last 10 minutes so you can get the seeds. Just pick them in any of the other two stages.

**Comment by jasonrohrer** [comment]
Good point, Twisted.

I will still make a way to clear them, just in case someone messes up.
